pkg/util/utils.go

```diff
@@ -156,6 +158,28 @@ func CalculateSubnetSize(mask int) int64 {
 }
 
 func IsSystemNamespace(c client.Client, ns string, obj *v1.Namespace) (bool, error) {
+	// first check t1 then vpc
+	isSysNs, err := IsT1Namespace(c, ns, obj)
+	if err != nil {
+		return false, err
+	}
+	if isSysNs {
+		return true, nil
+	}
+	// Only check VPC system namespace if VPC mode is enabled
+	if CoeConfig != nil && CoeConfig.CoeConfig.EnableVPCNetwork {
```

Comment:
I feel that you can just pass a vpcMode argument in IsSystemNamespace, so there is no need to introcuce a global var: CoeConfigin util.